src/main/java/ch/njol/skript/expressions/ExprChatRecipients.java

```diff
@@ -49,7 +49,7 @@
 @Name("Chat Recipients")
 @Description("Recipients of chat events where this is called.")
 @Examples("chat recipients")
-@Since("2.2 (unknown)")
+@Since("2.2-Fixes (unknown)")
```

Comment:
Are you sure Mirreski added this?

Reply:
Yes, the coding is unlike Njol, more of Mirre's style, and Chat Recipients was in RandomSk. It's an extremely high chance.

Reply:
He did add this, as you can see here:
https://github.com/Mirreski/Skript/commit/e5cb581df19c9e1bd3ea52db446fd757c3ca7763
it looks like it was in 2.2-Fixes-v7

Reply:
Will update to exact version.